book/extend-admin.rst

```diff
@@ -293,6 +293,12 @@ as well, then you should be able to see these actions when using the ``debug:rou
       app.put_event      PUT      ANY      ANY    /admin/api/events/{id}.{_format}
       app.delete_event   DELETE   ANY      ANY    /admin/api/events/{id}.{_format}
 
+.. note::
+
+    Routes to this actions must be exposed using `FOSJsRoutingBundle`_, and also they must have set ``methods``
+    parameter (e.g. ``options: ['expose' => true], methods: ['GET']``). Examples can be found at `Sulu Workshops`_, specificly in assignment 10.
```

Comment:
```suggestion
    The routes to these actions must be exposed using `FOSJsRoutingBundle`_, and also they must have set ``methods``
    parameter (e.g. ``options: ['expose' => true], methods: ['GET']``). Examples can be found in the `Sulu Workshops`_, specificly in assignment 10.
```

@alexander-schranz WDYT does this information already suffice or do you want it in more detail?